src/components/Project/ProjectBasics/ProjectForm/ProjectFormBanner.tsx

```diff
@@ -9,13 +14,50 @@ interface IProjectFormbannerProps {
 }
 const ProjectFormBanner: FC<IProjectFormbannerProps> = ({ onSubmit, isDirty }) => {
   const { t } = useTranslation();
+  const project = useAppSelector(selectProject);
+  const navigate = useNavigate();
+
+  const { isConfirmed } = useConfirmDialog();
```

Comment:
Maybe add an empty line between functions and variables to make it more readable.